packages/graalvm/src/main/kotlin/elide/runtime/plugins/vfs/VfsConfig.kt

```diff
@@ -16,24 +16,30 @@ package elide.runtime.plugins.vfs
 import java.net.URI
 import java.net.URL
```

Comment:
need copyright headers but we can add this later in a release commit, we should get your IDE configured to add them automatically.